wkconnect/backends/tiff/models.py

```diff
@@ -103,15 +112,27 @@ def clear_cache(self) -> None:
         self.read_properties.cache_clear()  # pylint: disable=no-member
         self.read_mmapped.cache_clear()  # pylint: disable=no-member
 
+    @staticmethod
+    def is_tiled(tif: TiffFile) -> bool:
+        tiled = False
+        for i, page in enumerate(tif.pages):
+            if (
+                TAG_TILE_WIDTH in page.tags
+                and TAG_TILE_HEIGHT in page.tags
+                and TAG_TILE_BYTE_OFFSETS in page.tags
+            ):
+                tiled = True
+        return tiled
```

Comment:
I guess this could also be
```suggestion
        for page in tif.pages:
            if (
                TAG_TILE_WIDTH in page.tags
                and TAG_TILE_HEIGHT in page.tags
                and TAG_TILE_BYTE_OFFSETS in page.tags
            ):
                return True
        return False
```

or even

```suggestion
        return any(
            TAG_TILE_WIDTH in page.tags
            and TAG_TILE_HEIGHT in page.tags
            and TAG_TILE_BYTE_OFFSETS in page.tags
            for page in tif.pages
        )
```

Reply:
yes, much better :) learning some python here, thanks for the suggestion!